Dockerfile.cli

```diff
@@ -1,6 +1,7 @@
 FROM php:7.0-cli-jessie
+RUN apt-get update && apt-get install -y git-core zip libxml2-dev
 RUN docker-php-ext-install pcntl
-RUN apt-get update && apt-get install -y git-core zip
+RUN docker-php-ext-install dom
```

Comment:
I'm not sure if this is used only in tests (PHPUnit requires it?) or whether it should be put everywhere